### PR TITLE
added Buttons with its variants

### PR DESCRIPTION
**Pull request by omjadhav85**
added Button components with its variants

image

### CODE REVIEW

File: styles/components/button.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+.btn-secondary {
+  border: 1px solid var(--primary-color);
+  background-color: white;
+  color: var(--primary-color);
+}
+
+.btn-link {
+  border: 1px solid var(--primary-color);
+  background-color: white;
```
Comment (lannisterrr) [inline styles/components/button.css]: make this color variable or use hex code ideally.

File: styles/components/button.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  color: var(--primary-color);
+}
+
+.btn-link a{
```
Comment (lannisterrr) [inline styles/components/button.css:30]: Idealy try to use specific selectors and not element selector as it can impact performance.

**Review by lannisterrr — APPROVED** [review]
Other than points mentioned above the code seems fine.

**Review by piyushsh25 — COMMENTED** [review]
LGTM👍

File: styles/components/button.css (hunk 1 above)
Comment (omjadhav85) [reply styles/components/button.css]: Made the changes..thanks!

**Pull request merged**